packages/space/src/index.ts

```diff
@@ -49,12 +51,33 @@ export default defineComponent({
       children.children.forEach((child: VNode, loopKey) => {
         if (isFragment(child)) {
           if (isArray(child.children)) {
+            // This inference is `ComputedRef`, It's actually wrong
+            const unrefItemStyle = unref(itemStyle)
+
+            const childLength = child.children.length - 1
+
             child.children.forEach((nested, key) => {
+              const nodeStype = {
+                paddingBottom: '0',
+                marginRight: '0',
+              }
+
+              if (childLength === key) {
+                if (direction === 'horizontal') {
+                  nodeStype.paddingBottom = unrefItemStyle.paddingBottom
+                } else {
+                  nodeStype.paddingBottom = '0'
+                }
+              } else {
+                nodeStype.marginRight = unrefItemStyle.marginRight
+                nodeStype.paddingBottom = unrefItemStyle.paddingBottom
+              }
+
```

Comment:
Actually here the type annotation is wrong, we should use `Unref<ReturnType<typeof useSpace>>` instead of `ReturnType<typeof useSpace>` here. The actual type of `itemStyle` is `Object` (despite the fact that this `itemStyle` is a computedRef, but it is a plain object here). We only need to mark the type. as I mentioned above.

Reply:
Refer to the image here:
![image](https://user-images.githubusercontent.com/15975785/110789353-bce35b80-82aa-11eb-9140-5a066955cc66.png)
![image](https://user-images.githubusercontent.com/15975785/110789392-ca98e100-82aa-11eb-866b-c00f09c5b92b.png)

Reply:
That was my bad.